editions/de-AT-server/tiddlywiki.info

```diff
@@ -2,7 +2,8 @@
 	"description": "Server configuration of the German (Österreich) edition",
 	"plugins": [
 		"tiddlywiki/tiddlyweb",
-		"tiddlywiki/filesystem"
+		"tiddlywiki/filesystem",
+		"tiddlywiki/highlight"
```

Comment:
The problem is, that html output is different to tiddlywiki.com. So if there are regressions in the core, they may be hidden by the plugin.